### PR TITLE
Feature responsive

### PR DESCRIPTION
**Pull request by wolf128058**
make the design more responsive:
- border-radius, margin-and padding, use short-hands
- error-message: with padding & font-weight
- no bottom padding for fieldset.gluon-section
- rem and not em, min partial is 1/8
- duplicates removed
- responsive display optimized: flex-displays and margins
- definitions joined and encapsulated -> this is scss not only css!
- initial viewport-scale and -width
- break-word for long titles and values (pubkey!)
- add .editorconfig-entry for scss-files

### CODE REVIEW

**Comment by NeoRaider** [comment]
This looks like lots of useful changes, but it would be much easier to review if each of the points you list in the commit description were a separate commit... I'll still have a short look and comment on a few things I notice.

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (NeoRaider) [inline package/gluon-config-mode-theme/sass/gluon.scss:73]: Missing semicolon

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (NeoRaider) [inline package/gluon-config-mode-theme/sass/gluon.scss:248]: These ID's can't be used for styling, they can change at any time. Which page/element does this even refer to?

File: .editorconfig
Comment (NeoRaider) [inline .editorconfig:40]: `tab` is already in the default entry `*`, which should cover everything that isn't listed explicitly. Also, there shouldn't be a reason to set an `indent_size` for tabs - after all, the main advantage of tab indentation is that every developer change change the tab width according to their own preferences. So as far as I can tell, this commit can be dropped entirely.

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (NeoRaider) [inline package/gluon-config-mode-theme/sass/gluon.scss:191]: The background color change doesn't seem to be covered by the changes listed in the commit/PR description.

**Review by NeoRaider — CHANGES_REQUESTED** [review]
I have noticed a few things, but as mentioned, I don't see how I can review this properly. It will definitely need to be split into separate commits. I also didn't actually try out the new version on my devices.

Please also follow the common scheme for commit message: Each commit title should be prefixed with the name of the package it changes (`gluon-config-mode-theme: ` in this case).

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss:191]: in another fork i used a bg-image. thats where this 0.333-opacity stuff is from. i'll take that one back.

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss:73]: you can actually remove the semicolon on the last attribute in {}.
but thats the only one in the complete file. so i will add a semicolon.

File: .editorconfig
Comment (wolf128058) [reply .editorconfig:40]: no problem. i will remove this completely.

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss]: fixed with f02b5f71278dfff70559172a2650b2c26e222945

File: .editorconfig
Comment (wolf128058) [reply .editorconfig]: fixed with 0f89c6b13b1f856b72c3b37e7198659cb85a3a30

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss]: fixed with 96f58d16511d4721ebcdee3be15872d808343810

File: package/gluon-config-mode-theme/sass/gluon.scss
Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss]: as far as i remeber atm: thats a textbox appearing next to the map.
because the osm-stuff is placed in another module. i remove that one completely.

Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss]: fixed with ff201e451e91052743b546b797e0206f776a54f0

Comment (wolf128058) [reply package/gluon-config-mode-theme/sass/gluon.scss]: ... and 4e66ee27f18518c6c0238023a9b4fc4132c756bd

Comment (NeoRaider) [reply package/gluon-config-mode-theme/sass/gluon.scss]: We should add CSS classes to the elements that need better styling, but we can leave that for a later PR.

**Comment by NeoRaider** [comment]
Thanks for the fixes.

I don't know how familiar you are with Git, but if possible, the commits should still be reorganized (you can replace the PR using a force push after squashing and separating commits).

**Review by AiyionPrime — CHANGES_REQUESTED** [review]
First of all thanks for your contribution @wolf128058, UI design fixes are about as rare as appreciated.

I haven't got much experience in webdesign, but am doing pretty well with git, so far.
If you'd like, we could meet up online for an hour or two of peer _programming_, where you'd explain to me, what different sections of your scss changes are doing and I'd put them into commits accordingly.

If you'd rather do this on your own, that would be fine as well, but currently this is not going anywhere near merging, which would be a shame regarding the work you've put into it already.

An answer would be appreciated either way, if this stays silent until new year, I'd close this, as code-dumps are not :/

Thanks again and have a nice weekend,
Aiyion

**Comment by AiyionPrime** [comment]
@wolf128058 If you want to get this moving again, just mention me, or reply to my mail and we can get this done in an afternoon, or so.

**Comment by maurerle** [comment]
I tested this and think that we should merge this (after fixing the color):

copying the above gluon.css into `/lib/gluon/config-mode/www/static/gluon.css` this is the result:
image
image

instead of the old one:
image
image


I think this is a huge benefit which should not be just closed.

**Comment by AiyionPrime** [comment]
@maurerle I like his changes as well. The form he submitted was just not reviewable and after a year of waiting on any answer, as well as the reminder a month ago which was left unanswered I think it was fair to close it, to make room for other contributors, like yourself.

If you want to start working on this, I suggest you checkout the commits and rebase it on your own branch and open a new PR referencing this.
As long as you work with his commits, make sure to be a good sport and use a "Co-authored-by: " line, so that you both can be credited for this.

Looking forward to your contribution :)

**Pull request closed without merging**
